lib/ui/widgets/post_content/widgets/link_preview/index.dart

```diff
@@ -22,9 +23,9 @@ class _LinkPreviewState extends State<LinkPreview> {
   void initState() {
     super.initState();
     fetchPreview(widget.urls).then((RichLinkPreview res) {
-      setState(() {
-        data = res;
-      });
+      if (mounted) {
+        setState(() => data = res);
+      }
```

Comment:
just curious where does mounted come from? :O

Reply:
@ryuash It's a property of the `State` class: [`mounted` property](https://api.flutter.dev/flutter/widgets/State/mounted.html) :grin: